worldedit/code.lua

```diff
@@ -3,16 +3,22 @@
 
 --- Executes `code` as a Lua chunk in the global namespace.
 -- @return An error message if the code fails, or nil on success.
-function worldedit.lua(code)
-	local func, err = loadstring(code)
-	if not func then  -- Syntax error
-		return err
+function worldedit.lua(code, name)
+	local factory, err = loadstring("return function(p) " .. code .. " end")
+	if not factory then  -- Syntax error
+		return false, err
 	end
-	local good, err = pcall(func)
-	if not good then  -- Runtime error
-		return err
+	local func=factory()
+	local player=minetest.get_player_by_name(name)

```

Comment:
when no name is given this calls `get_player_by_name(nil)`.
Even if this doesn't cause an error it should be avoided.